R/scale.R

```diff
@@ -110,6 +110,20 @@ prep.step_scale <- function(x, training, info = NULL, ...) {
 
   sds <- sds * x$factor
 
+  which_sd <- which(sds ==0)
```

Comment:
It doesn't work great in all situations to compare to zero with a numeric value like this because of how floating point numbers are stored. Instead, you can see if it is less than the machine tolerance with `sds < .Machine$double.eps` or `all.equal()` or something like that.

Reply:
I didn't know about that, I should have paid more attention to your original comment my bad. I will implement it.

Reply:
Done